plugins/org.python.pydev.debug/src_console/org/python/pydev/debug/newconsole/env/PydevIProcessFactory.java

```diff
@@ -169,19 +170,24 @@ public PydevConsoleLaunchInfo createInteractiveLaunch() throws UserCanceledExcep
             }
 
             if (interpreter == null) {
```

Comment:
At the start of this method, please change the code to get the active editor from:

```
        IWorkbenchWindow workbenchWindow = PlatformUI.getWorkbench().getActiveWorkbenchWindow();
        IWorkbenchPage activePage = workbenchWindow.getActivePage();
        IEditorPart activeEditor = activePage.getActiveEditor();
```

to use:

`org.python.pydev.shared_ui.EditorUtils.getActiveEditor()`

-- the current code could fail with a NullPointerException if there's no active page.